src/lib/intent/editorinterface-reset.ts

```diff
@@ -0,0 +1,41 @@
+import Intent from './base-intent'
+import { PlanMessage } from '../interfaces/plan-message'
+import { ResetEditorInterfaceAction } from '../action/editorinterface-reset'
+import chalk from 'chalk'
+import { SaveEditorInterfaceAction } from '../action/editorinterface-save'
+
+export default class EditorInterfaceResetIntent extends Intent {
+  isEditorInterfaceReset () {
+    return true
+  }
+  groupsWith (other: Intent): boolean {
+    const sameContentType = other.getContentTypeId() === this.getContentTypeId()
+    return other.isEditorInterfaceReset() && sameContentType
+  }
+  endsGroup (): boolean {
+    return true
+  }
+  shouldSave (): boolean {
+    return false
+  }
+  shouldPublish (): boolean {
+    return false
+  }
+  toActions () {
+    // TODO: check for optimizing API calls and saving multiple editor interfaces changes on the same content type done as 1 single API call instead of multiple.
```

Comment:
That's what `groupsWith` does.
Everything that `groupsWith` returns true for will be turned into a single request later on.
You will ave to set `endsGroup` to `false` though